src/views/LeadAdd/LeadDetail/index.js

```diff
@@ -169,605 +149,571 @@ const defaultProps = {
     organizations: [],
 };
 
-// FIXME: change to functional component
-class LeadDetail extends React.PureComponent {
-    static propTypes = propTypes;
-
-    static defaultProps = defaultProps;
-
-    constructor(props) {
-        super(props);
+function LeadDetail(props) {
+    const {
+        activeProject,
+        assignees,
+        bulkActionDisabled,
+        className: classNameFromProps,
+        confidentialityOptions,
+        disableLeadUrlChange,
+        lead,
+        leadGroups,
+        leadState,
+        onApplyAllBelowClick,
+        onApplyAllClick,
+        onChange,
+        onOrganizationsAdd,
+        onLeadGroupsAdd,
+        organizations,
+        pending: pendingFromProps, // pending lead options
+        priorityOptions,
+    } = props;
+
+
+    const [searchedText, setSearchedText] = useState(undefined);
+
+    const options = useMemo(
```

Comment:
Rename to `organizationSearchRequestOptions` or something similar but has to be specific